### ISSUE TITLE
Sending waypoints and missions with mavlink

### ISSUE THREAD

**Issue by hsrfeyzullah** [padcom/node-mavlink#9]
Hello, I am working on a mission planner. I am successful in assigning different waypoints on the map and sending them to my boarding pass, but I cannot start a mission. Can you help with this?

**Comment by padcom** [comment]
I'm sorry, this might be out of my league. Check out the docs for Mavlink protocol, maybe there is something that needs to be done before you can start the mission.

In any case, if you'll encounter any problems sending or receiving messages, feel free to open up another ticket. Thanks!
